content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -318,7 +318,7 @@ Currently, storage size is the only resource that can be set or requested.  Futu
 
 {{< feature-state for_k8s_version="v1.13" state="beta" >}}
 
-Prior to Kubernetes 1.9, all volume plugins created a filesystem on the persistent volume.
+Prior to Kubernetes 1.9, all volume plugins created a filesystem on the `PersistentVolume`.
 Now, you can set the value of `volumeMode` to `block` to use a raw block device, or `filesystem`
```

Comment:
I don't think you need to talk about how things were before in an unsupported version. Please rephrase.